test/integration/inject/inject_test.go

```diff
@@ -30,67 +35,197 @@ func TestMain(m *testing.M) {
 /// TEST EXECUTION ///
 //////////////////////
 
+func parseDeployment(yamlString string) (*appsv1.Deployment, error) {
+	s := json.NewYAMLSerializer(json.DefaultMetaFactory, scheme.Scheme,
+		scheme.Scheme)
+	var deploy appsv1.Deployment
+	_, _, err := s.Decode([]byte(yamlString), nil, &deploy)
+	if err != nil {
+		return nil, err
+	}
+
+	return &deploy, nil
+}
+
 func TestInjectManual(t *testing.T) {
```

Comment:
@alpeb I wonder whether it makes sense to have this here. We are pretty much testing the same logic as in `TestInjectManualParams` but with a limited set of config params

Reply:
Agreed, I don't see it adding value.